src/ui/window/BrowserWindow.qml

```diff
@@ -502,7 +502,7 @@ ApplicationWindow {
                     'loadContent': false,
                     'url': restoreTabs[i].url,
                     'title': restoreTabs[i].title,
-                    'iconUrl': restoreTabs[i].iconUrl
+                    'iconUrl': restoreTabs[i].icon.replace("https", "http")
```

Comment:
This bothers me. Right now QML Image is not able to load images via https, showing SSL error in console. Yet browser works fine with SSL. Don't know how to fix it right now.

Reply:
Interesting, I cannot reproduce this issue here on Ubuntu 17.10 with Qt 5.10.1.